commons-executors/src/main/java/com/palantir/common/concurrent/PTExecutors.java

```diff
@@ -623,8 +662,12 @@ protected Runnable wrap(Runnable runnable) {
         };
     }
 
+    /**
+     * Wraps the given {@code ExecutorService} so that {@link ExecutorInheritableThreadLocal}
+     * variables are propagated through.  This does not create a span on the thread.
+     */
     public static ExecutorService wrap(final ExecutorService executorService) {
-        return wrap("PTExecutor", executorService);
+        return Tracers.wrap(executorService);
```

Comment:
Why are we changing what we wrap our executors with?

Reply:
(This changes the behavior of `newCachedThreadPoolWithMaxThreads` and `newFixedThreadPool`)

Reply:
Was looking at the wrong `wrap` method 🤦🏼

Reply:
OK cool -- I think we should instead just remove this `PTExecutors.wrap` method, and simply just reference `Tracers.wrap`.

Reply:
As to answer `Why are we changing what we wrap our executors with?`, it appears that we use inheritable thread-local variables to pass tracing information down onto threads which are spawned through the parent thread (so that they all have the same trace). Seeing as we're no longer doing spans for these threadpools, this seems ok to me. I'd like another atlas contributor to verify that this is, in fact, OK.

Reply:
Actually, come to think of it, this change may not be safe to make. Seeing as TimedRunner is used to effectively run every KVS call, are we sure we will pass the span to any threads spawned through the TimedRunner? 

My ask here is we make a unit test to prove that this is the case.

Reply:
I agree with @Sam-Kramer (2 points above); we do not need the threadLocal context for executor with no span. 
But also agree that changing the existing method might be unsafe.
This `wrap` method is public, even though it is not being used in AtlasDb, it could be in use internally. We can just make this change additively i.e. add a new method `wrapWithoutSpan` and have this change there.

Reply:
We should certainly add a unit test in for `newCachedThreadPoolNoSpan` in PtExecutorsTest.
I would also encourage we cut an rc and deploy on internal AtlasDb proxy and verify traces do not include the the spans removed by this PR.

Reply:
So the reason I thought it would be safe to make this change, even as a public method, is that any code that hits this method is going to create a span with the name "PTExecutor" and have no metadata, so be virtually worthless anyway.

I am happy to make wrapWithoutSpan instead of changing this, but i expect that we don't want anyone hitting the pre-changed code anyway.

Reply:
`are we sure we will pass the span to any threads spawned through the TimedRunner?`

Yes, that is what Tracers.wrap does.  Happy to make a test verifying it though

Reply:
ok, there is a lot in this one.  Here are the actions I think I have:

- [ ] make a new method called wrapWithoutSpan
- [ ] restore old wrap method
- [ ] unit test for newCachedThreadPoolNoSpan to verify it has no span, but has thread context passed along
- [ ] cut RC and deploy on a test proxy (will need help or directions for this one)

If I have omitted something, please flag and I will add.